src/AbstractValidator.php

```diff
@@ -653,7 +653,7 @@ protected function isTime($value, string $format = "g:i A"): bool
    *
    * @return bool
    */
-  protected function isDate($value, $format = "m/d/Y"): bool
+  protected function isDate($value, string $format = "m/d/Y"): bool
```

Comment:
also just fixing what should have been corrected before; my IDE noticed the lack of a parameter type hint so I put it in.